types/gl-react-dom/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "files": [
+    "index.d.ts"
+  ],
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "noImplicitAny": true,
+    "noImplicitThis": false,
+    "strictNullChecks": true,
+    "strictFunctionTypes": false,
```

Comment:
Any reason `noImplicitThis` and `strictFunctionTypes` are off?

Reply:
No reason, except I tried to match the only "dependency" (e.g. `gl-react`) tsconfig. I enabled now `noImplicitThis` and `strictFunctionTypes`.